res/values-pt-rBR/strings.xml

```diff
@@ -262,16 +262,16 @@
     <string name="Clan">"Clã"</string>
 
 
-    <string name="JOIN_CLAN">"ENTRAR CLÃ"</string>
-    <string name="JOIN_RANDOM_CLAN">"ENTRAR EM CLÃ ALEATÓRIO"</string>
+    <string name="JOIN_CLAN">"ENTRAR NO CLÃ"</string>
+    <string name="JOIN_RANDOM_CLAN">"ENTRAR EM UM CLÃ ALEATÓRIO"</string>
     <string name="LEAVE_CLAN">"SAIR DO CLÃ"</string>
     <string name="INVITE_MEMBER">"CONVIDAR MEMBRO"</string>
     <string name="REMOVE_MEMBER">"REMOVER MEMBRO"</string>
 
 
-    <string name="CREATE_CLAN">"CRIAR CLÃ"</string>
+    <string name="CREATE_CLAN">"CRIAR UM CLÃ"</string>
     <string name="MEMBER">"MEMBRO"</string>
-    <string name="ADMIN">"ADMIN"</string>
+    <string name="ADMIN">"ADMINISTRADOR"</string>
```

Comment:
Please keep it as "ADMIN", because players are already used to it and Admin can be also understandable since the word for Administrator in portuguese also starts with "Admin".